### PR TITLE
OP-1339 Fix lots overlooking when they reach zero quantity

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1339.
Paired with https://github.com/informatici/openhospital-core/pull/1441.

Also add workflow refactoring

### CODE REVIEW

**Comment by mwithi** [comment]
going to fix build upon "push" events too...

**Comment by mwithi** [comment]
@dbmalkovsky now should be ok

File: bundle/language_en.properties [hunk 1]
```diff
@@ -829,10 +829,13 @@ angal.medicalstock.multiplecharging.chargetype
 angal.medicalstock.multiplecharging.chooseamedical                                                     = Choose a Medical
 angal.medicalstock.multiplecharging.cost                                                               = Cost
 angal.medicalstock.multiplecharging.datecannotbebeforelastmovementdate.msg                             = Date cannot be before last movement date.
+angal.medicalstock.multiplecharging.empty                                                              = Empty
```
Comment (dbmalkovsky) [inline bundle/language_en.properties:832]:
```suggestion
angal.medicalstock.multiplecharging.empty.txt                                                          = Empty
```
Technically all message constants should have some kind of suffix: txt, btn, key, msg, fmt.msg, etc

See `expired` and `hideemptylots` below for other examples.

**Review by dbmalkovsky — APPROVED** [review]
LGTM (given you ignore my suffix suggestion).

File: bundle/language_en.properties (hunk 1 above)
Comment (mwithi) [reply bundle/language_en.properties:832]: I realize now the all the other bundles around have no suffix... but in this case what suffix we should apply?

**Review by ArnaudFofou — APPROVED** [review]

File: bundle/language_en.properties (hunk 1 above)
Comment (mwithi) [reply bundle/language_en.properties:832]: not sure about the suffix to use, we will do a revision in another issue.

**Pull request merged**